src/components/BalanceRow.tsx

```diff
@@ -81,6 +81,7 @@ export const BalanceRow = ({
               id={`${assetString}-actions`}
               onChange={(e) => onAction(e.target.value, asset)}
               disabled={disabled}
+              value=""
```

Comment:
I acknowledge this is very simplistic :D , but if we only care about this dropdown defaulting to the initial value on re-render, this will do the trick. 

I *could* save the dropdown's state and then update after `onAssetAction`'s completion, but that seems like overkill as this appears to be work for all the use cases I tried. Am I missing a scenario where we would need the added flexibility of saving in state?

Reply:
We might want to check with the Product if this is acceptable because in this case user will never see the action selected (re-render happens when we show the modal).

![image](https://user-images.githubusercontent.com/7346473/114555732-fe628000-9c35-11eb-8a34-70c49ead15e5.png)

Reply:
🤔 Maybe we could reset the value only when it is not `isActive`.

Reply:
Ah good catch - I knew the dropdown was resetting when we show the modal, but on my screen the modal was always covering the dropdown. I'll put in a better fix!

Reply:
That is the desired behavior. Thanks for pointing that out @quietbits.